rviz_tactile_plugins/src/tactile_visual_base.h

```diff
@@ -137,9 +139,11 @@ protected Q_SLOTS:
 
   ::tactile::Range raw_range_;
   RangeProperty *range_property_;
+  BoolProperty *disable_range_update_property_;
```

Comment:
I suggest moving the property down into the RangeProperty.

Reply:
Now that you say, I think that it is the comment you said last time and I had forgotten. 
If the property displays in a similar way to my proposal and does not make more complex a "global" checkbox, why not.